docs/I18N.md

```diff
@@ -27,7 +27,7 @@ With each browser page refresh, the client initialization process includes parsi
 <a name="import"></a>
 ## How to import a new .po file into the front-end i18n scheme so it's available as a selection in the client.
 
-When adding a new .po file, keep in mind that the actual name of the file prior to the `.po` (eg: _Bocce_ from **Bocce.po**) is what will be used to dynamically populate the choice of languages that can be selected in the client via the settings page. Capitalization will be carried over, so be sure to name your .po file exactly as you'd like it applied as a language choice.
+When adding a new .po file, keep in mind that the actual name of the file prior to the `.po` (eg: _Latin_ from **Latin.po**) is what will be used to dynamically populate the choice of languages that can be selected in the client via the settings page. Capitalization will be carried over, so be sure to name your .po file exactly as you'd like it applied as a language choice.
```

Comment:
Suggestion for your consideration:

When adding a new .po file, remember that the file's name prior to the `.po` (eg: _Latin_ from **Latin.po**) will be used to dynamically populate the language choices in the client's settings page. Capitalization will be carried over, so be sure to name your .po file exactly as you'd like to see it as a language choice.